pkg/config/config_template.yaml

```diff
@@ -23,7 +24,8 @@ api_key:
 
 # Force the hostname to whatever you want. (default: auto-detected)
 # hostname: mymachine.mydomain
-
+{{ end }}
+{{- if .Agent }}
```

Comment:
dogstatsd use goexpvar. we should move `expvar_port` the common part. Same for `forwarder_timeout`, `forwarder_retry_queue_max_size`. `histogram_aggregates` and `histogram_percentiles`.

Reply:
dsd has its own `dogstatsd_stats_port` that got lost, re-added and moving the other settings